docs/upgradeFromRC3.md

```diff
@@ -217,6 +210,7 @@ Important here is the `STATUS` column. It **needs** to be a number that goes up
 ### Step 3:
 
 Instead of using the provided `composer.lock` out of the box we are going to loosen certain dependencies and bring manually Archipelago modules, all this to make update easier and future upgrades less of a pain.
```

Comment:
We provide now a composer.lock.default. We also need to be sure that the user doing this runs all composer require against the original *(their)* composer.lock and not against the new default one

Reply:
And does this make sense? 8a7361c1df8a1b7098b0ba2eb1d063b5d429ff04